concent_api/utils/logging.py

```diff
@@ -220,3 +219,50 @@ def get_subtask_id_for_logging(message):
     if not isinstance(subtask_id, str):
         subtask_id = ''
     return subtask_id
+
+
+def log_gatekeeper_endpoint_request_received(request, operation: str):
+    path_to_file = request.META['PATH_INFO'] if 'PATH_INFO' in request.META.keys() else '-path to file UNAVAILABLE-'
+    logger.info("{} request received. File '{}'".format(
+        operation,
+        path_to_file
+    ))
+
+
+@replace_element_to_unavailable_instead_of_none
+def log_gatekeeper_message_under_validation(operation_type: str, file_path: str, subtask_id: bytes, public_key: bytes):
+    logger.info("{} request will be validated. File '{}', with subtask_id '{}'. Client public key: '{}'".format(
+        operation_type.capitalize(),
+        file_path,
+        subtask_id,
+        public_key
+    ))
+
+
+@replace_element_to_unavailable_instead_of_none
+def log_gatekeeper_message_successful_validated(operation: str, file_path: str, subtask_id: bytes, public_key: bytes):
```

Comment:
Should be called `log_message_under_validation`, and message type should be logged as well